apps/site/components/Downloads/ReleaseModal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import AlertBox from '@node-core/ui-components/Common/AlertBox';
+import Modal from '@node-core/ui-components/Common/Modal';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import { MinorReleasesTable } from '@/components/Downloads/MinorReleasesTable';
+import LinkWithArrow from '@/components/LinkWithArrow';
+import type { NodeRelease } from '@/types';
+import { getReleaseAnnounceLink } from '@/util/getReleaseAnnounceLink';
+
+type ReleaseModalProps = {
+  isOpen: boolean;
+  closeModal: () => void;
+  release: NodeRelease;
+};
+
+const ReleaseModal: FC<ReleaseModalProps> = ({
+  isOpen,
+  closeModal,
+  release,
+}) => {
+  const t = useTranslations();
+
+  const modalHeadingKey = release.codename
+    ? 'components.releaseModal.title'
+    : 'components.releaseModal.titleWithoutCodename';
+
+  const modalHeading = t(modalHeadingKey, {
+    version: release.major,
+    codename: release.codename ?? '',
+  });
+
+  const releaseAnnounceLink = getReleaseAnnounceLink(release);
+
+  return (
+    <Modal open={isOpen} onOpenChange={closeModal} heading={modalHeading}>
```

Comment:
The modal should not cover the whole page. It should be a way smaller modal since the table itself also only has two columns xD

Reply:
(Im referring to the width ofc)